policy/release/lib/attestations.rego

```diff
@@ -14,7 +14,9 @@ taskrun_att_build_types := [
 	"https://tekton.dev/attestations/chains@v2",
 ]
 
-hacbs_test_task_result_name := "HACBS_TEST_OUTPUT"
+hacbs_test_task_result_name := "TEST_OUTPUT"
```

Comment:
Update to not has `hacbs` in the variable name?

Reply:
Done.